package.json

```diff
@@ -37,6 +38,7 @@
     "react-apollo-hooks": "^0.5.0",
     "react-app-rewired": "^2.2.1",
     "react-dom": "^18.2.0",
+    "react-hook-form": "^7.38.0",
```

Comment:
Please utilise the validation and Form provided with antd : https://ant.design/components/form